TemplateDoesNotExist at /admin-dashboard/cars/add/
admin/car_form.html
Request Method:	GET
Request URL:	http://11ff9cc2-bf81-4a7c-a3fb-c9090db2cffc-00-2rv27od5g692g.pike.replit.dev/admin-dashboard/cars/add/
Django Version:	5.2
Exception Type:	TemplateDoesNotExist
Exception Value:	
admin/car_form.html
Exception Location:	/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/django/template/loader.py, line 19, in get_template
Raised during:	rental.admin_views.wrapper
Python Executable:	/nix/store/wqhkxzzlaswkj3gimqign99sshvllcg6-python-wrapped-0.1.0/bin/python3
Python Version:	3.11.10
Python Path:	
['/home/runner/workspace',
 '/home/runner/workspace/.pythonlibs/bin',
 '/nix/store/lc7qv2ldzrs1aq3hbyzmbgvn0h2w26pl-sitecustomize/lib/python/site-packages',
 '/nix/store/dfg63lldbvcj207iyr9z0xw21d8ax02n-python3.11-pip-24.0/lib/python3.11/site-packages',
 '/nix/store/clx0mcir7qw8zk36zbr4jra789g3knf6-python3-3.11.10/lib/python311.zip',
 '/nix/store/clx0mcir7qw8zk36zbr4jra789g3knf6-python3-3.11.10/lib/python3.11',
 '/nix/store/clx0mcir7qw8zk36zbr4jra789g3knf6-python3-3.11.10/lib/python3.11/lib-dynload',
 '/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages',
 '/nix/store/clx0mcir7qw8zk36zbr4jra789g3knf6-python3-3.11.10/lib/python3.11/site-packages',
 '/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages',
 '/nix/store/wblvmd5y7izx0z10d1w7ga7zc4apjxmb-python3.11-setuptools-75.1.1/lib/python3.11/site-packages']
Server time:	Sun, 06 Apr 2025 02:11:17 +0000
Template-loader postmortem
Django tried loading these templates, in this order:

Using engine django:

django.template.loaders.filesystem.Loader: /home/runner/workspace/templates/admin/car_form.html (Source does not exist)
django.template.loaders.app_directories.Loader: /home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/django/contrib/admin/templates/admin/car_form.html (Source does not exist)
django.template.loaders.app_directories.Loader: /home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/django/contrib/auth/templates/admin/car_form.html (Source does not exist)
django.template.loaders.app_directories.Loader: /home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/crispy_bootstrap5/templates/admin/car_form.html (Source does not exist)
Traceback Switch to copy-and-paste view
/home/runner/workspace/.pythonlibs/lib/python3.11/site-packages/django/core/handlers/exception.py, line 55, in inner